### PR TITLE
[FXLA11Y] What is an accessible FXL EPUB?

### PR DESCRIPTION
**Pull request by wareid**
Adding some text to explain what an accessible ebook is. For discussion at next meeting.

+ Fixing some formatting issues

### CODE REVIEW

**Review by Jeffxz — APPROVED** [review]

**Comment by gregoriopellegrino** [comment]
If it were possible I would like to refer to the guidelines we have available. What do you think of a sentence like:

> An accessible Fixed Layout EPUB is an EPUB that follows the EPUB 3 specification for the production of Fixed Layout publications (https://www.w3.org/TR/epub-33/#sec-fixed-layouts) and is compliant with EPUB Accessibility (https://w3c.github.io/epub-specs/epub33/a11y/) whose content therefore meets the Web Content Accessibility Guidelines (https://www.w3.org/WAI/standards-guidelines/wcag/).

Perhaps the sentence is a bit wordy; it can be improved.

Also, I would like to add something like:

> Creating accessible EPUB Fixed Layouts does not require pre-recorded audio (Media Overlay), the presence of which, while not mandatory, may help some readers.

**Comment by artbyrt** [comment]
I'm hoping I can find an answer to an issue that is occurring in reading systems (Thorium, Books in Mac) that shouldn't be happening. The book/s are fxl comic books using a separate xhtml page for the extended image description with link from image page and backlink from description page. In the .opf spine, the extended description page is listed as linear="no". But, the readers are rolling over into these pages instead of stopping at the official *last* page. An older version of Thorium actually behaved correctly and did not display the extended description pages but a recent update has removed that. Any suggestions as to fixes would be welcome.

**Comment by mattgarrish** [comment]
> But, the readers are rolling over into these pages instead of stopping at the official _last_ page.

Setting a document as non-linear doesn't mean that it won't be presented to users when they page through the content. It just means that it doesn't have to be read linearly where it's placed (e.g., reading systems could present it where it is, skip it, or move it to the end of the spine; all are valid options).

There have been numerous discussions about whether skipping non-linear document should be the default presentation, but we haven't been able to reach a consensus. Some people want to be able to reach every document without having to click on hyperlinks to get to them.

Consequently, there is no "fix" to what you're encountering other than to ask the developers for the option to have an alternative presentation that hides non-linear content.
